### ISSUE TITLE
Support for searching non-Dutch terms in AAT

### ISSUE THREAD

**Issue by EnnoMeijers** [netwerk-digitaal-erfgoed/network-of-terms-catalog#75]
Currently the aat.rq search query requires prefLables to be defined as Dutch using the following non-optional clause:
`?uri skosxl:prefLabel ?prefLabel_uri .`
`?prefLabel_uri dcterms:language aat:300388256 . # Dutch (language)`

This leads to the problem that non-Dutch labels which are commonly used are not being found e.g. "alba amicorum'. This is a Latin term that is commonly used for book categories but has no Dutch prefLabel which is probably correct because it is a Latin term. Is there a way to support finding these terms without flooding the results with hits from many different languages?

**Comment by ddeboer** [comment]
The resource indeed has no dutch `skos:prefLabel`, but they seem to have no hesitations about ‘translating’ the Latin term, because it does have a Dutch `skos:altLabel` one. Moreover, it does have `skos:prefLabel` in languages other than Latin, albeit not in Dutch:

```ttl
aat:300374942
    skos:prefLabel "alba amicorum"@en , "alba amicorum"@la , "albums amicorum"@fr , "雜品集"@zh-hant , "zá pǐn jí"@zh-latn-pinyin-x-hanyu , "za pin ji"@zh-latn-pinyin-x-notone , "tsa p'in chi"@zh-latn-wadegile , "Stammbücher (alba amicorum)"@de ;
    skos:altLabel "alba amicorum"@nl , "album amicorum"@fr , "album amicorum"@en, "album amicorum"@la , "album   amicorum"@nl , "libri amicorum"@en , "libri amicorum"@la , "liber amicorum"@en , "liber amicorum"@la , "albums amicorum"@en , "Albums amicorum"@de , "Stammbuch (album amicorum)"@de , "Album amicorum"@de ;
```

So while I think this should be solved in the source itself, by adding a Dutch prefLabel just like they already have an altLabel, we could fall back to English for the prefLabel.

**Comment by EnnoMeijers** [comment]
I agree adding a Dutch prefLabel would be the right thing to do. As far as I know this process could take a long before this change really ends up in Getty's endpoint. I prefer to have a fall back for situations like these. We might also consider to search in both prefLabel and altLabel as we do with other sources as well e.g. cht.rq. If that doesn't result in any results we could fall back to English.

**Comment by raboof** [comment]
Also agree this should be fixed at the Getty side, but until then also looking at the altLabel sounds like a good idea.

Also, for cases like CoBra (http://vocab.getty.edu/aat/300411283) which do not have Dutch labels at all, it would be great if we could fall back to the English prefLabel - though I'm not sure how easy it is to express "does not have a Dutch label" in SPARQL?
